processing/src/main/java/io/druid/query/DefaultGenericQueryMetricsFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import io.druid.jackson.DefaultObjectMapper;
+
+public class DefaultGenericQueryMetricsFactory implements GenericQueryMetricsFactory
+{
+  private static final DefaultGenericQueryMetricsFactory INSTANCE = new DefaultGenericQueryMetricsFactory(new DefaultObjectMapper());
```

Comment:
It probably doesn't matter much, since the ObjectMapper is only used to serialize the query context as JSON. There won't be anything in there that requires fancy Jackson modules to be loaded.

Maybe worth doing if it's easy, just for cleanliness. But I think it's not essential.